test/suite/test_verbose01.py

```diff
@@ -58,13 +52,16 @@ def expect_verbose(self, categories, patterns):
         # operations on the connection (for generating the expected verbose output).
         yield conn
         # Read the contents of stdout to extract our verbose messages.
-        output = self.readStdout(self.nlines)
+        nlines = 30000
```

Comment:
Any reasons for moving this definition as a local variable?
I'd prefer to leave this definition as a field in the `test_verbose_base` class, allowing any class that extends `test_verbose_base` to override it (based on their needs)

Reply:
My only motivation was that I always try to reduce any variable's scope, I thought we could extend its scope only when we need it.
But I am happy to revert to anticipate that need!

Reply:
As discussed, I reverted the changes 5d5ef9f2f27d5d1c3fb60b8d5677785558bacc29.